scripts/zones/Jugner_Forest/Zone.lua

```diff
@@ -14,6 +14,9 @@ zoneObject.onInitialize = function(zone)
     UpdateNMSpawnPoint(ID.mob.FRAELISSA)
     GetMobByID(ID.mob.FRAELISSA):setRespawnTime(math.random(900, 10800))
 
+    UpdateNMSpawnPoint(ID.mob.METEORMAULER)
+    GetMobByID(ID.mob.METEORMAULER):setRespawnTime(math.random(900, 10800))
```

Comment:
Interesting.  I would expect CI to trip on this because theoretically GetMobByID can return nil, but it can't for static mobs like this. Just leaving a comment here for Clay to see (you don't need to do anything until CI complains about it)